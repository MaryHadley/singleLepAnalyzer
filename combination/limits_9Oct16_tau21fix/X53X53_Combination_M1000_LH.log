000 (1295375.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295375.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.233:40949>
...
006 (1295375.000.000) 10/09 22:41:34 Image size of job updated: 51812
	8  -  MemoryUsage of job (MB)
	7968  -  ResidentSetSize of job (KB)
...
006 (1295375.000.000) 10/09 22:46:34 Image size of job updated: 283696
	145  -  MemoryUsage of job (MB)
	147848  -  ResidentSetSize of job (KB)
...
006 (1295375.000.000) 10/10 00:01:42 Image size of job updated: 283696
	145  -  MemoryUsage of job (MB)
	147860  -  ResidentSetSize of job (KB)
...
006 (1295375.000.000) 10/10 18:38:41 Image size of job updated: 297968
	158  -  MemoryUsage of job (MB)
	160816  -  ResidentSetSize of job (KB)
...
005 (1295375.000.000) 10/10 18:47:10 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 20:00:33, Sys 0 00:02:56  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 20:00:33, Sys 0 00:02:56  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31727        1   3824814
	   Memory (MB)          :      158     3072      3072
...
