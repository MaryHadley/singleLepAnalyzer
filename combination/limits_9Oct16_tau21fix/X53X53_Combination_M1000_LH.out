/var/lib/condor/execute/dir_28919
Total number of likelihood evaluations: 1100000100
Total number of likelihood evaluations: 11000100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_LH/quant-toys:0-sig-9aca437465.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_LH/quant-data-sig-d5f992fb04.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1000_LH
