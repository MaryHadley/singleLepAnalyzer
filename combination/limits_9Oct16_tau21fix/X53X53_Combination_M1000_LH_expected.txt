# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.088929   0.044853     0.1849  0.061451     0.1303
